css/index.css

```diff
@@ -1,3 +1,452 @@
+}
+
+.content-row .middle-img {
+	width: 100%;
+}
+
+.content-row section {
+	padding-right: 10px;
+}
+
+@media screen and (max-width: 800px) {
+    .content-row img {
+        height: 150px;
+        width: 100%;
+        margin-top: 15px;
+    }
+    .content {
+        padding-bottom: 45px;
+    }
+}
+
+@media screen and (max-width: 500px) {
+	.content {
+		padding-top: 40px;
+	}
+	.content-row {
+		flex-direction: column;
+		margin-top: 20px;
+		margin-bottom: 0px;
+		padding: 0px 10px;
+	}
+
+	.content section{
+		width: 100%;
+		margin-bottom: 150px;
+		height: 0px;
+	}
+
+	.content-row img {
+		margin-top: -15px;
+		margin-bottom: 11px;
+		height: 96px;
+	}
+
+}
+
+/* End of Content */
+
+/* Footer */
+.footer {
+	width: 100%;
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	justify-content: flex-start;
+	padding-bottom: 100px;
+}
+
+.footer p {
+	padding-top: 10px;
+}
+
+.copyright {
+	align-self: center;
+	margin: 20px 0px;
+}
+
+@media screen and (max-width: 800px) {
+    .footer {
+        margin-top: 6px;
+        margin-left: 18px;
+    }
+
+    .copyright {
+        margin-top: 40px;
+    }
+}
+
+/* End of footer */
+
+
+/* Services HTML */
+
+.services-main {
+	display: flex;
+	flex-direction: column;
+	align-items: flex-start;
+	margin: 10px 0px;
+	padding: 30px 0px;
+}
+
+.services-main h1{
+	font-size: 35px;
+}
+
+.container .services-header-img{
+	width: 100%;
+}
+
+
+/* Service Content CSS */
+
+.service-content {
+	width: 100%;
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-between;
+	border-bottom: solid black 2px;
+	border-top: solid black 2px;
+	padding: 20px 0px;
+}
+
+.service-content-box {
+	width: 430px;
+	height: 250px;
+	padding: 30px;
+	margin: 20px 0px;
+	border: solid 1px black;
+	background: rgb(245, 245, 245);
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+	position: relative;
+}
+
+.service-content button {
+	width: 100px;
+	height: 30px;
+	border: solid 1px black;
+	border-radius: 5px;
+	position: absolute;
+	bottom: 30px;
+	left: 30px;
+}
+
+
+/* End of Service Content CSS */
+
+/* Facts About us CSS */
+
+.facts-about {
+	display: flex;
+	justify-content: space-between;
+	padding: 50px 0px;
+}
+
+.facts-about div {
+	display: flex;
+	flex-direction: column;
+	justify-content: flex-start;
+}
+
+.facts-about p {
+	padding: 10px 70px 20px 0px;
+}
+
+
+.facts-about ul{
+	padding-left: 20px;
+	margin-top: 20px;
+}
+
+.facts-about ul li{
+	list-style-type: disc;
+	margin-bottom: 20px;
+}
+
+/* End of Facts About us CSS */
+
+
+/* End of Services HTML */
+
+
+/* Media Queries */
+@media screen and (max-width: 800px) {
+	.service-content {
+		align-items: center;
+	}
+
+	.service-content-box {
+		height: 300px;
+		margin: 20px auto;
+	}
+
+	.facts-about {
+		flex-direction: column;
+	}
+
+	.facts-about img {
+		width: 100%;
+		height: auto;
+	}
+
+	.facts-about p {
+		width: 100%;
+		padding: 10px 0px 20px 0px;
+	}
+
+
+}
+
+/* End of Media Queries */
```

Comment:
Very well done on this! you'll get even more responsive when you begin percentage-based margins and borders.